runtime/build.rs

```diff
@@ -23,7 +23,11 @@ fn main() {
         // time, one to get the metadata and te recompile it with the metadata hash in an environment
         // variable.
         #[cfg(feature = "metadata-hash")]
-        let builder = builder.enable_metadata_hash("ACME", 18);
+        let builder = if std::env::var_os("ZKV_FORCE_DISABLE_METADATA_HASH").is_none() {
```

Comment:
Any reason why you put this here in this PR ?

Reply:
Should be in a separate PR. I've inserted it just to speedup my develop process because I used to have all features enable to spot out issue also in the code branches gated in by other features.

Have all features enable also means compile the wasm code twice every time, so I add an env variable that every one can add to its `~/.cargo/config.toml` file and disable the doble compilation.

If you prefer I can rid it out and open a separate PR